package.json

```diff
@@ -9,6 +9,9 @@
             "packages/*"
         ]
     },
+    "resolutions": {
+        "zod": "3.22.5"
```

Comment:
is there another package using zod ?

Reply:
I don't thin it's necessary anymore, removed.